app/app.rb

```diff
@@ -0,0 +1,86 @@
+require 'sinatra/base'
+require 'json'
+
+class FakeIdvaas < Sinatra::Application
+  get '/' do
+    'Hello, USA!'
+  end
+
+  get '/idvaas/user/:user_uuid/question/?' do
```

Comment:
Ahh, I see what you were doing (http://www.sinatrarb.com/faq.html#slash). Good call!